packages/blazor-workspace/NimbleBlazor/Components/NimbleWaferMap.razor.cs

```diff
@@ -118,6 +123,23 @@ public partial class NimbleWaferMap : ComponentBase
         }
     }
 
+    /// <summary>
+    /// Represents the input data, an apache arrow Table, which fills the wafer map with content
+    /// </summary>
+    [Parameter]
+    public RecordBatch? DiesTable
```

Comment:
So, I think we need to take the `async` pattern all the way, and do away with this property-based API for setting data on the `WaferMap`, and transition to the `async` method-based approach as we have with the `Table`.